### PR TITLE
Update privacy and terms of conditions

### PR DESCRIPTION
**Pull request by eugene-eko2000**

### CODE REVIEW

File: be/server.go [hunk 1]
```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	EndPointUser   = "/update_or_create_user"
-	EndPointReport = "/report"
-	EndPointGetMap = "/get_map"
+	EndPointUser          = "/update_or_create_user"
+	EndPointPrivacyAndTOC = "/update_privacy_and_toc"
+	EndPointReport        = "/report"
+	EndPointGetMap        = "/get_map"
```
Comment (eldarm) [inline be/server.go:15]: If you are trying to make it in alphabetic order, then GetMap goes first and user last.

File: be/user.go [hunk 2]
```diff
@@ -45,3 +53,37 @@ func UpdateUser(c *gin.Context) {
 	}
 	c.Status(http.StatusOK) // 200
 }
+
+func UpdatePrivacyAndTOC(c *gin.Context) {
```
Comment (eldarm) [inline be/user.go:57]: I thought we agreed to have one file per endpoint, no?

**Review by eldarm — COMMENTED** [review]
Overall ok, with two comments.
Plus, do we really need that privacy thing? Since we don't collect PII at all.

File: be/server.go (hunk 1 above)
Comment (eugene-eko2000) [reply be/server.go:15]: It's in a functionality order. User and Privacy are grouped together.

File: be/user.go (hunk 2 above)
Comment (eugene-eko2000) [reply be/user.go:57]: I moved it into its own file

**Comment by eugene-eko2000** [comment]
> Overall ok, with two comments. Plus, do we really need that privacy thing? Since we don't collect PII at all.

We need it, the user has to agree with terms of conditions anyway. And there is an option if to display avatars or show reports fully anonymously..

**Pull request closed without merging**